callstats-java-sdk/src/main/java/io/callstats/sdk/CallStats.java

```diff
@@ -225,5 +242,142 @@ public synchronized void sendCallStatsBridgeStatusUpdateFromQueue() {
 			sendCallStatsBridgeStatusUpdate(bridgeStatusInfo);
 		}
 	}
+
+	public synchronized void sendCallStatsConferenceEvents(CallStatsConferenceEvents eventType, ConferenceInfo conferenceInfo,
```

Comment:
are we sending 1 or multiple event? If 1 perhaps we rename it to "sendCallStatsConferenceEvent"?